tests/atlocal.in

```diff
@@ -76,6 +76,9 @@ COBC="${COBC} -fdiagnostics-plain-output"
 COMPILE="${COBC} -x ${FLAGS}"
 COMPILE_ONLY="${COBC} -fsyntax-only ${FLAGS} -Wno-unsupported"
 COMPILE_MODULE="${COBC} -m ${FLAGS}"
+LISTING_FLAGS="-fttitle=GnuCOBOL_V.R.P -fno-ttimestamp"
```

Comment:
Shouldn't `-t -` be in there, too? For `AT_CHECK we still can override this by `-T -` for the tests that check the wide listing.
I'll leave that decision to you, just wanted to bring it up.

Reply:
There are some places where we still need to output to a file. I will keep it outside for now.